cli/src/main/kotlin/com/malinskiy/marathon/cli/args/FileIOSConfiguration.kt

```diff
@@ -35,6 +35,7 @@ data class FileIOSConfiguration(
         @JsonProperty("keepAliveIntervalMillis") val keepAliveIntervalMillis: Long = 0L,
         @JsonProperty("deviceInitializationTimeoutMillis") val deviceInitializationTimeoutMillis: Long?,
         @JsonProperty("devices") val devices: File?,
+        @JsonProperty("disableHostKeyVerifier") val disableHostKeyVerifier: Boolean = false,
```

Comment:
For now this is an okay solution, but we need to think of a way to do the proper verification later

Reply:
the verification is on by default this flag just allows to disable it for some cases. 
as for the verification on its own, it's done on `SSHj` side and I'm not sure if we can enhance it from within the marathon.